boards/FSM/Inc/App/App_FsmWorld.h

```diff
@@ -29,6 +29,7 @@ struct FsmWorld *App_FsmWorld_Create(
     struct InRangeCheck *     steering_angle_in_range_check,
     struct InRangeCheck *     brake_pressure_in_range_check,
     struct BinaryStatus *     brake_actuation_status,
+    struct BinaryStatus *     brake_oc_sc_status,
```

Comment:
could we rename this to `brake_open_circuit_or_short_circuit`? Just to be a bit more clear.

Reply:
Ken addressed this earlier. I renamed it to brake_open_or_short_circuit.